theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,319 @@
+      <div class="settings-group">
+        <h3>Java Arguments</h3>
+        <input
+          v-model="settings.custom_java_args"
+          type="text"
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
+        />
+      </div>
+      <div class="settings-group">
+        <h3>Environment Arguments</h3>
+        <input
+          v-model="settings.custom_env_args"
+          type="text"
+          class="input installation-input"
+          placeholder="-Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled"
+        />
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <div class="sliders">
+          <span class="slider">
+            Minimum Memory
+            <Slider v-model="settings.memory.minimum" :min="1024" :max="8192" :step="1024" />
+          </span>
+          <span class="slider">
+            Maximum Memory
+            <Slider v-model="settings.memory.maximum" :min="1024" :max="8192" :step="1024" />
+          </span>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Window Size</h2>
+      <div class="settings-group">
+        <div class="settings-group">
+          <div class="sliders">
+            <span class="slider">
+              Width
+              <Slider v-model="settings.game_resolution[0]" :min="400" :max="2562" :step="2" />
+            </span>
+            <span class="slider">
+              Height
+              <Slider v-model="settings.game_resolution[1]" :min="400" :max="2562" :step="2" />
+            </span>
+          </div>
+        </div>
+        <hr class="card-divider" />
+        <div class="settings-group">
+          <h3>Console</h3>
+          <div class="toggle-setting">
+            Maximum Concurrent Downloads
+            <input
+              v-model="settings.max_concurrent_downloads"
+              type="text"
+              name="concurrent-downloads"
+              class="concurrent_downloads"
+            />
+          </div>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Commands</h2>
+      <div class="settings-group">
+        <div class="toggle-setting">
+          Pre Launch
+          <input ref="javaArgs" v-model="settings.hooks.pre_launch" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Wrapper
+          <input ref="javaArgs" v-model="settings.hooks.wrapper" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Post Launch
+          <input ref="javaArgs" v-model="settings.hooks.post_exit" type="text" class="input" />
+        </div>
+      </div>
+    </Card>
   </div>
 </template>
+
+<style lang="scss">
+.save-btn {
+  position: absolute !important;
+  z-index: 100;
+  top: 4rem;
+  right: 4rem;
+  opacity: 0;
+  transition: 0.2s ease-in-out all;
+}
```

Comment:
This save button placement isn't in line with our styling. I would put it at the top right of the theme container or just make it save as items change